repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -3,10 +3,13 @@
 include::_includes/in-progress.adoc[]
 
 == Future Release
+* https://repose.atlassian.net/browse/REP-4465[REP-4465] - Internal clean up converting some enums to constants.
```

Comment:
This belongs on master not here, yes we'll have to deal with the merge, but i'd rather do that then have someone accidentally drop it into the release notes for a version on this branch.